uconnrcmpy/conditions.py

```diff
@@ -851,6 +851,15 @@ def compare_to_sim(self, run_reactive=False, run_nonreactive=True):
             print_str += '{:.6f}'.format(ignition_delay)
             copy_str += '{}'.format(ignition_delay)
 
+        if self.nonreactive_sim is not None and self.reactive_sim is not None:
+            # Use np.where to find the index of EOC for reactive simulation
+            # Use the given index to find T_EOC_reactive
+            # Compare to T_EOC (nonreactive case) using np.isclose
+            EOC_reactive_idx = np.where(self.reactive_sim.time >= compression_time)
+            T_EOC_reactive = self.reactive_sim.temperature[EOC_reactive_idx][0]
+            if not np.isclose(T_EOC, T_EOC_reactive, rtol = 0.001):
+                print('T_EOC_reactive =', T_EOC_reactive, 'K', '\n', 'T_EOC_nonreactive =', T_EOC, 'K', '\n', 'WARNING: Temperatures at Reactive EOC and non-reactive EOC did not match')
```

Comment:
One more change. I'd like to limit the line lengths to less than 100 characters, so we need to change this line a little bit. Let's use 3 print statements, since each print statement will automatically cause a newline to be printed when it runs. Then, we can use the special `format` syntax of strings to fill in the values

```
print('WARNING: Temperatures at Reactive EOC and non-reactive EOC did not match')
print('T_EOC_nonreactive = {} K'.format(T_EOC))
print('T_EOC_reactive = {} K'.format(T_EOC_reactive))
```